### PR TITLE
Changed Name and added expanded Abstract

### PR DESCRIPTION
**Pull request by torgo**
An attempt to write a new, more friendly, abstract and also rename the document as discussed in this [week's virtual f2f](https://github.com/w3ctag/meetings/blob/gh-pages/2020/05-distributed/README.md).

This should be reflecting the discussion in:
https://github.com/w3ctag/design-principles/issues/191

The following issue has been closed:
https://github.com/w3ctag/design-principles/issues/153

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/pull/197.html" title="Last updated on May 29, 2020, 1:20 AM UTC (bfd8667)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/197/049ce03...bfd8667.html" title="Last updated on May 29, 2020, 1:20 AM UTC (bfd8667)">Diff</a>

### CODE REVIEW

**Comment by annevk** [comment]
Isn't this #191? What about the suggestions there?

**Comment by torgo** [comment]
Yep. This got out of sync for some reason. I will fix.

**Comment by plinss** [comment]
With the addition of the section on CSS, @dbaron and I were wondering if the term 'API' is still appropriate in the title (without getting into philosophical debates about what an 'API' is).

This carries over beyond the title to other areas, e.g. perhaps drop the 'API' from section 2 and section 9 headers, s/API/Feature/ in section 2.4, ...

**Comment by torgo** [comment]
“Web Platform Feature Design Principles”?

File: index.bs [hunk 1]
```diff
@@ -9,7 +9,7 @@ ED: https://w3ctag.github.io/design-principles/
 Editor: Sangwhan Moon, Invited Expert, https://sangwhan.com
 Former Editor: Domenic Denicola, Google https://www.google.com/, https://domenic.me/, d@domenic.me
 Former Editor: Travis Leithead, Microsoft, travil@microsoft.com
-Abstract: This document contains a small-but-growing set of design principles collected by the W3C TAG while <a href="https://github.com/w3ctag/spec-reviews/">reviewing</a> specifications.
+Abstract: This document contains a set of design principles to be used when designing Web Platform technologies. These principles have been collected during the Technical Architecture Group's discussions in href="https://github.com/w3ctag/spec-reviews/">reviewing</a> developing specifications. We encourage specification designers to read this document and use it as a resource when making design decisions. We suggest that specification designers read <a href="#basic-principles">section 1</a> in full and use the remainder of the document as a reference.
```
Comment (hober) [inline index.bs:12]: This looks great. My only nit is the missing `<a`.

**Comment by hober** [comment]
Other possibilities:

* "Web Platform Design Principles"
* "Client-side Web Platform Design Principles"
* "Design principles for web platform features"
* "Design principles for web features"

**Comment by annevk** [comment]
Yeah, my suggestion was Web Platform Design Principles, i.e., "Platform" instead of "API". I see now that it was ambiguous.

**Comment by hober** [comment]
Whoops, I definitely overwrote your last change, @torgo. Fixing now.

File: index.bs [hunk 2]
```diff
@@ -9,7 +9,7 @@ ED: https://w3ctag.github.io/design-principles/
 Editor: Sangwhan Moon, Invited Expert, https://sangwhan.com
 Former Editor: Domenic Denicola, Google https://www.google.com/, https://domenic.me/, d@domenic.me
 Former Editor: Travis Leithead, Microsoft, travil@microsoft.com
-Abstract: This document contains a small-but-growing set of design principles collected by the W3C TAG while <a href="https://github.com/w3ctag/spec-reviews/">reviewing</a> specifications.
+Abstract: This document contains a set of design principles to be used when designing Web Platform technologies. These principles have been collected during the Technical Architecture Group's discussions in <a href="https://github.com/w3ctag/spec-reviews/">reviewing</a> developing specifications. We encourage specification designers to read this document and use it as a resource when making design decisions. We suggest that specification designers read <a href="#basic-principles">section 1</a> in full and use the remainder of the document as a reference.
```
Comment (alice) [inline index.bs:12]:
```suggestion
Abstract: This document contains a set of design principles to be used when designing Web Platform technologies. 
These principles have been collected during the Technical Architecture Group's discussions in 
<a href="https://github.com/w3ctag/spec-reviews/">reviewing</a> 
developing specifications. 
We encourage specification designers to read this document 
and use it as a resource when making design decisions.
```

Comment (alice) [inline index.bs:12]:
```suggestion
Abstract: This document contains a set of design principles to be used when designing Web Platform technologies. These principles have been collected during the Technical Architecture Group's discussions in <a href="https://github.com/w3ctag/spec-reviews/">reviewing</a> developing specifications. We encourage specification designers to read this document and use it as a resource when making design decisions.
```

**Pull request merged**